### PR TITLE
[14.0][MIG] account_move_force_removal module

### PR DESCRIPTION
**Pull request by tafaRU**

### CODE REVIEW

File: account_move_force_removal/tests/test_move.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+                "code": "test-sale-jorunal",
+                "company_id": cls.env.company.id,
+            }
+        )
+        cls.product = cls.env["product.product"].create(
+            {"name": "Test product", "type": "service"}
+        )
+        cls.income_account = cls.env["account.account"].search(
+            [("user_type_id.name", "=", "Income")], limit=1
+        )
```
Comment (victoralmau) [inline account_move_force_removal/tests/test_move.py]:
```suggestion
       account_type = self.env.ref("account.data_account_type_other_income")
       cls.income_account = cls.env["account.account"].search(
            [("user_type_id", "=", account_type.id)], limit=1
        )
```

**Review by victoralmau — CHANGES_REQUESTED** [review]

File: account_move_force_removal/tests/test_move.py (hunk 1 above)
Comment (tafaRU) [reply account_move_force_removal/tests/test_move.py]: Thanks for the suggestion! Done.

**Review by victoralmau — APPROVED** [review]

**Review by pedrobaeza — APPROVED** [review]
You can squash administrative commits into the first real one (from v13).

**Comment by tafaRU** [comment]
Thanks for the review @pedrobaeza 

About:
> You can squash administrative commits into the first real one (from v13).

It seems it's already done.

**Comment by pedrobaeza** [comment]
No, it isnt':

Selección_002

**Pull request merged**